pages/api/extension/relevant.ts

```diff
@@ -18,11 +18,23 @@ const relevantHandler = async (req: NextApiRequest, res: NextApiResponse) => {
 		res.status(200).send("Ok");
 		return;
 	}
+
+	const event_properties = {
+		repo_provider: req.body.repo_provider || "",
+		repo_owner: req.body.repo_owner || "",
+		repo_name: req.body.repo_name || "",
+	}
 	// For normal requests
 	if (!("repo_provider" in req.body) ||
 		!("repo_owner" in req.body) ||
 		!("repo_name" in req.body) ||
 		!("user_id" in req.body)) {
+			const eventProperties = { ...event_properties, response_status: 401 };
+			rudderStackEvents.track("absent", "", 'chrome_extension_relevant-handler', {
+				type: 'relevant-prs',
```

Comment:
type can be HTTP-401

Reply:
If @avikalpg is not already using these events. If that's true, we can keep the naming